### ISSUE TITLE
Relative option doesn't work

### ISSUE THREAD

**Issue by corlletelab** [spatools/grunt-html-build#49]
It seem that the relative option for styles and scripts does not work. Compared to the 0.3.1 version there is a changed line in  /grunt-html-build/tasks/build-html.js Line 134:
```javascript
var url = options.relative ? path.relative(path.dirname(options.dest), f) : f;
````

If I revert this to:
```javascript
var url = options.relative ? path.relative(options.dest, f) : f;
```
then paths are generated correctly.

**Comment by timhettler** [comment]
Cab you create a pull request for this?

**Comment by timhettler** [comment]
Submitted fix: #50

**Comment by SomaticIT** [comment]
Fixed in __0.4.2__.
